docs/iris/src/userguide/interpolation_and_regridding.rst

```diff
@@ -117,41 +133,90 @@ coordinate is unevenly spaced in altitude:
        723.58   790.31   863.41   942.88  1028.74  1120.98  1219.61]
 
 We could regularise the vertical coordinate by defining 10 equally spaced altitude
-sample points between 400 and 1250:
+sample points between 400 and 1250 and interpolating our vertical coordinate onto
+these sample points:
 
     >>> sample_points = [('altitude', np.linspace(400, 1250, 10))]
     >>> new_column = column.interpolate(sample_points, iris.analysis.Linear())
     >>> print new_column.summary(shorten=True)
     air_potential_temperature / (K)     (model_level_number: 10)
 
-To see what is going on, let's look at the original data, the interpolation line, and
-the new data in a plot: 
+Let's look at the original data, the interpolation line and
+the new data in a plot. This will help us to see what is going on:
 
 .. plot:: userguide/regridding_plots/interpolate_column.py
 
-As we can see with the red diamonds on the extremes of the altitude values, we have
+The red diamonds on the extremes of the altitude values show that we have
 extrapolated data beyond the range of the original data. In some cases this is desirable
-functionality, and for others it is not - for instance, this column defined
-a surface altitude value of 414m, so extrapolating "air potential temperature" at 400m
-in this case makes little physical sense.
-
-Fortunately we can control the extrapolation mode when defining the interpolation scheme
-with the ``extrapolation_mode`` keyword.  For :class:`iris.analysis.Linear` the
-``extrapolation_mode`` must be one of ``linear``, ``error``, ``nan``, ``mask`` or
-``nanmask``. To mask the values which lie beyond the range of the original data, using
-the ``mask`` extrapolation mode is just a matter of constructing the appropriate scheme
-and passing it through to the :meth:`~iris.cube.Cube.interpolate` method:
+functionality but in other cases it is not. For example, this column defines
+a surface altitude value of 414m, so extrapolating an "air potential temperature" at 400m
+makes little physical sense in this case.
+
+We can control the extrapolation mode when defining the interpolation scheme.
+Controlling the extrapolation mode means we can avoid situations like the above where
```

Comment:
I disagree because it adds a pronoun ("this"; bad news) and removes the physical grounding of the point this paragraph is making (also bad news, IMO).